spec/absolute.html

```diff
@@ -329,9 +329,8 @@ <h1>Temporal.Absolute.prototype.toLocaleString ( [ _locales_ [ , _options_ ] ] )
     </emu-clause>
 
     <emu-clause id="sec-temporal.absolute.prototype.tojson">
-      <h1>Temporal.Absolute.prototype.toJSON ( [ _timeZone_ ] )</h1>
+      <h1>Temporal.Absolute.prototype.toJSON ( )</h1>
```

Comment:
Hmm, I'm guessing it should be used, though.

Reply:
I'll just remove this commit from this PR for now, and open an issue to figure it out.

Reply:
→ #446